backend/climateconnect_api/views/donation_views.py

```diff
@@ -16,6 +16,12 @@
 
 
 class GetDonationGoalProgress(APIView):
+    """
```

Comment:
Keeping it so we can deploy and fall back if needed. In a next release can then remove this endpoint.